ref/commands/server/server-windowsservice.adoc

```diff
@@ -6,74 +6,88 @@
 //
 // Contributors:
 //     IBM Corporation
+:page-layout: server-command
+:page-type: command
+= server WinService command
```

Comment:
I don't like this title because it doesn't really make sense to me. This isn't a WinService command, it is a set of 4 related commands that are only valid on windows. I think perhaps the title below might be better:

    = Manage Liberty as a Windows service